docs/primitives/a-box.md

```diff
@@ -9,7 +9,7 @@ section_order: 5
 source_code: src/extras/primitives/primitives/meshPrimitives.js
 ---
 
-The box primitive creates shapes such as boxes, cubes, or walls.
+The box primitive creates a quadrilateral using the [geometry component] with type set to `box`.
```

Comment:
I think this is more technical and confusing than the original for beginners. Also quadrilateral is usually applied for 2D shapes.